packages/eslint-plugin/src/configs/react-config.js

```diff
@@ -0,0 +1,136 @@
+const configHelpers = require('../utils/configHelpers');
+
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  plugins: ['jsx-a11y', 'react', 'react-hooks', '@griffel'],
```

Comment:
rebased to preserve griffel plugin addition introduced in https://github.com/microsoft/fluentui/pull/22961

cc @layershifter pls take a look, also note I'm not using recommended config set rather direct rule approach, as from experience adding some pre-set configuration might make it hard for inheritance as various rules need to be explicitly turn off ( this flaw is already present within our eslint config unfortunately )

Reply:
That works, I can push updates once we will have more rules 👍